demo/performance/lit-html.html

```diff
@@ -4,7 +4,11 @@
     <meta charset="utf-8">
     <!-- We use “unsafe-eval” below because we need to trick the browser into
           not caching tagged template string objects to test interpretation. -->
-    <meta http-equiv="content-security-policy" content="default-src 'self'; script-src 'self' 'unsafe-eval' 'sha256-2gPcHEAV/bRWIxMnz3UA5z6w2m9RdFis8ZAa8Y/EZVg=' https://esm.sh/lit-html@3.2.1 https://esm.sh/lit-html@3.2.1/;">
+    <meta http-equiv="content-security-policy" content="
+      default-src 'self';
+      script-src 'self' 'unsafe-eval' 'sha256-2gPcHEAV/bRWIxMnz3UA5z6w2m9RdFis8ZAa8Y/EZVg='
+      https://esm.sh/lit-html@3.2.1
+      https://esm.sh/lit-html@3.2.1/;">
```

Comment:
I was one-lining this stuff because any other formatting choice seemed wonky. I’m totally fine with this for now, but I think we might want to take a moment to discuss the conventions here 🤙